src/display/general/overallGraphs.go

```diff
@@ -206,7 +235,12 @@ func RenderCharts(ctx context.Context,
 			}
 
 		case <-tick: // Update page with new values
-			updateUI()
+			if !helpVisible {
+				ui.Render(myPage.Grid)
+				for i := 0; i < numCores; i++ {
+					ui.Render(myPage.CPUCharts[i])
+				}
+			}
```

Comment:
This could be shortened to a call to `updateUI()` itself right?

Reply:
It could, but it would cause unneccesary re-renders when the help menu is shown (since that does not need to be updated on every tick)

Reply:
I can change it to updateUI there if you prefer, should not cause any issues